rclpy/src/rclpy/action_client.hpp

```diff
@@ -205,15 +205,15 @@ class ActionClient : public Destroyable, public std::enable_shared_from_this<Act
    *        is_result_response_ready)
    */
   py::tuple
-  is_ready(WaitSet & wait_set);
+  is_ready(rclpy::WaitSet & wait_set);
```

Comment:
The function implementations in ``action_client.cpp`` still have the signatures without ``rclpy::`` namespace. Maybe we can change those to match the ones in headers ?

Reply:
https://github.com/ros2/rclpy/blob/b9ed8c96e002c1cf8da839cffdb0b63b6a28b375/rclpy/src/rclpy/action_client.cpp#L244